SmartDeviceLink/SDLIAPSession.m

```diff
@@ -139,6 +273,51 @@ - (SDLStreamErrorHandler)streamErroredHandler {
     };
 }
 
+- (SDLStreamHasSpaceHandler)streamHasSpaceHandler {
+    __weak typeof(self) weakSelf = self;
+    
+    return ^(NSStream *stream) {
+        __strong typeof(weakSelf) strongSelf = weakSelf;
+        
+        if (strongSelf.isDataSession){
+            SessionSendHandler sendCompletion;
+            @synchronized (strongSelf) {
+                sendCompletion = strongSelf.sendBlock;
+            }
+            
+            if (sendCompletion != nil){
+                NSError *err = nil;
+                BOOL sendComplete = sendCompletion(&err);
+                if (err != nil){
+                    NSLog(@"ERROR!!! Output stream reported %@", err);
```

Comment:
> Based on how frequently you are able to reproduce the issue, I am open to making the PR a hotfix or a standard bug fix.

So that you know "hotfix" and "bug fix" are identical, we only differentiate change types by [semantic versioning](https://semver.org).

The delay is necessary due to attempts to resume apps failing in certain cases. I haven't had to think about it in a while, so I'm not exactly sure. @asm09fsu do you remember?